Classes/NodeCreationHandler/NodeCreationHandlerInterface.php

```diff
@@ -17,6 +17,7 @@
 /**
  * Contract for Node Creation handler that allow to hook into the process just before a node is being added
  * via the Neos UI
+ * @api
```

Comment:
to be transparent with you ;) that is the only php class/interface in the neos.ui which deserves an `@api` notation. Any objections? Or do you need another thing to be declared `@api` as well?